modules/hackathon-2023/Hackathon2023FAQ.tsx

```diff
@@ -1,12 +1,162 @@
+        marginBottom: theme.spacing(8),
+      },
+    },
+    gridContainer: {
+      marginBottom: theme.spacing(4),
+    },
+    typographyH2: {
+      fontSize: '2rem',
+      marginBottom: theme.spacing(2),
+      '$gridContainer:first-of-type &': {
+        color: lighten(theme.palette.secondary.light, 0.05),
+      },
+      '$gridContainer:nth-of-type(2) &': {
+        color: lighten(theme.palette.highlight.light, 0.05),
+      },
+      '$gridContainer:nth-of-type(3) &': {
+        color: lighten(theme.palette.lowlight.light, 0.05),
+      },
+      [theme.breakpoints.up('md')]: {
+        textAlign: 'right',
+        paddingRight: theme.spacing(4),
+        marginBottom: theme.spacing(6),
+      },
+    },
+    typographyParagraph: {
+      marginBottom: theme.spacing(2),
+    },
+  })
+);
 
 const Hackathon2023FAQ = (): ReactElement => {
+  const classes = useStyles();
   return (
-    <div>
-      Hackathon2023Footer
-    </div>
+    <section className={classes.root} id="contest-outline">
+      <div className={classes.container}>
+        <Typography
+          className={classes.typographyOverline}
+          variant="overline"
+          component="h1"
+        >
+          Frequently Asked Questions
+        </Typography>
+
+        <Grid container className={classes.gridContainer}>
+          <Grid item md={5}>
+            <TypographyHeading>
+              What&rsquo;s a Hackathon?
+            </TypographyHeading>
+          </Grid>
+          <Grid item md={7}>
+            {
+              BODY_HACKATHON.map((copy, index) => (
+                <TypographyBody key={index}>
+                  {copy}
+                </TypographyBody>
+              ))
+            }
+          </Grid>
+        </Grid>
+
+        <Grid container className={classes.gridContainer}>
+          <Grid item md={5}>
+            <TypographyHeading>
+              What&rsquo;s InQueeries?
+            </TypographyHeading>
+          </Grid>
+          <Grid item md={7}>
+            {
+              BODY_INQUEERIES.map((copy, index) => (
+                <TypographyBody key={index}>
+                  {copy}
+                </TypographyBody>
+              ))
+            }
+          </Grid>
+        </Grid>
+
+        <Grid container className={classes.gridContainer}>
+          <Grid item md={5}>
+            <TypographyHeading>
+              What&rsquo;s involved?
+            </TypographyHeading>
+          </Grid>
+          <Grid item md={7}>
+            {
+              BODY_PARTICIPANT_TASKS.map((copy, index) => (
+                <TypographyBody key={index}>
+                  {copy}
+                </TypographyBody>
+              ))
+            }
+          </Grid>
+        </Grid>
+      </div>
+    </section>
   );
 };
 
-export default Hackathon2023FAQ;
+type CustomTypographyProps = {
+  children: ReactNode,
+};
 
+const TypographyHeading = ({ children }: CustomTypographyProps): ReactElement => {
+  const classes = useStyles();
```

Comment:
I suppose instead of using the same `useStyles`, we could make a new `useStyles` object to just include the `typographyH2` class? I don't think that matters in terms of performance or whatever so I think just having the one `useStyles` is better, just thinking out loud here. 

Note that a lot of my PR comments are conversational in nature, so this is me just leaving a comment here haha.

Reply:
I like this approach! Great idea. I will update my PR :) (PS. I love all feedback, even if it isn't performance and simply cleaning up the code style!)

Reply:
Updated - had to override an `any` type though, didn't have enough time to figure out the correct typing 😅